api/vehicles.go

```diff
@@ -115,3 +115,36 @@ func (api *API) UpdatesHandler(w http.ResponseWriter, r *http.Request) {
 	// Convert updates to JSON
 	WriteJSON(w, updates) // it's good to take some REST in our server :)
 }
+
+// HistoryHandler updates for last 30 days to endpoint
```

Comment:
In the spirit of improving documentation, it may be worthwhile to elaborate on this a bit, maybe something like "HistoryHandler returns the last 30 days worth of updates for all enabled vehicles" I only mention this because the way this is written, if a vehicle is disabled its historical updates won't be returned.

Reply:
Will update now thanks for the feedback!